collections/_article/godotfest-2025.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "A new convention to look forward to: GodotFest - Save the Date!"
+excerpt: "Introducing the second annual Godot convention: GodotFest!"
+categories: ["events"]
+author: GodotFest Team
+date: 2025-06-15 00:00:00
+
+image: /storage/blog/covers/godotfest-2025.webp
+---
+
+We’re excited to announce a new annual, independent convention centered around Godot: **GodotFest**!
+
+What does this mean? It means we’ll now have another annual Godot event to look forward to! Godot*Con*, still organized
+by the Godot Foundation, will continue to travel the world, offering a great starting point for anyone curious about
+the engine.
+
+Godot*Fest*, meanwhile, will be held in the same location each year, always around the same time of the year. Since it’s aimed more
+at a professional audience, it’ll take place during the workweek to make business travel more convenient.
+
+While not quite the same as GodotCon, GodotFest is organized by the Munich Godot user group,
+the same team behind the GodotCon 23. That's why it will take place in **Munich, Germany,
+on the 11th and 12th of November, at Smartvillage Bogenhausen**.
+
+Next year, you can expect to have at least three events: GodotCon North America, GodotCon Europe and GodotFest. 
```

Comment:
Not sure if it should call out three distinct ones right now? Also, @coppolaemilio and I discussed to also say "GodotFest 25 - GodotCon Europe"

Reply:
well, that is going to be the reality next year, but for this year I think the post already makes clear that the only event left is the GodotFest